backend/isoko-backend/src/handlers/review/get-user-reviews.js

```diff
@@ -34,28 +34,36 @@ exports.getUserReviewsHandler = async (event) => {
       },
    };
 
-   const dynamoResult = await docClient.query(params).promise();
+   let response;
 
-   let queryResults = dynamoResult.Items;
-   console.log(queryResults);
+   try {
+      const dynamoResult = await docClient.query(params).promise();
+      let queryResults = dynamoResult.Items;
+      console.log(queryResults);
```

Comment:
I know this isn't from your pr, but can we either get rid of this or change it to a console.info with a bit more information about what is being logged